restart-prod.sh

```
@@ -1 +1 @@
-git pull && docker compose down && docker compose up -d --build --remove-orphans
\ No newline at end of file
+git pull && docker compose down -f compose.beam.yml  && docker compose up -d --build --remove-orphans -f compose.beam.yml
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)